sdk/src/beta9/abstractions/mixins.py

```diff
@@ -0,0 +1,48 @@
+from typing import Callable, Optional
+
+from .. import terminal
+from ..clients.gateway import DeployStubRequest, DeployStubResponse
+from ..config import ConfigContext
+from .base.runner import RunnerAbstraction
+
+
+class DeployableMixin:
+    func: Callable
+    parent: RunnerAbstraction
+    deployment_stub_type: str
+
+    def _validate(self):
+        if not hasattr(self, "func") or not isinstance(self.func, Callable):
+            raise AttributeError("func variable not set or is incorrect type")
+        if not hasattr(self, "parent") or not isinstance(self.parent, RunnerAbstraction):
```

Comment:
nit pick but can we space this out a bit. kind of hard to read

Reply:
Updated.